media_remover/src/main/scala/MediaNotRequiredMessageProcessor.scala

```diff
@@ -55,24 +55,40 @@ class MediaNotRequiredMessageProcessor(asLookup: AssetFolderLookup)(
     })
 
 
+  def getMd5ChecksumForOnline(vsItemId: String): Future[Option[String]] =
+    getOriginalShapeForVidispineItem(vsItemId).map({
+      case Left(err) =>
+        logger.info(s"Couldn't get MD5 checksum for vsItemId $vsItemId: $err")
+        None
+      case Right(originalShape) => originalShape.getLikelyFile.flatMap(_.md5Option)
+    })
+
+
   def getVidispineSize(itemId: String): Future[Either[String, Option[Long]]] =
```

Comment:
In this branch: 

In case of `handleInternalArchiveCompleteForOnline`, gives a `Future(Left(s"Could not get online size from Vidispine, retrying"))`.
In case of `handleDeepArchiveCompleteOrReplayedForOnline`, gives `throw new RuntimeException(s"fileSize is missing")`

Maybe they should both return `Left`s?

However, these are not yet wired up in the `ProcessorConfiguration`:s in `Main`

Reply:
Probably `Lefts` for both. We could look at a self-healing loop for this case later on